### ISSUE TITLE
Add a "test your installation" in the documentation 

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#843]
### Motivation for the feature

It would be comforting for the user to have a section "test your installation" after documenting how to install ivadomed. Maybe we can suggest to run pytest?

**Comment by dyt811** [comment]
Yes, that is a great point. On the road map, we really hope to convert the few tutorials into "semi automated full integration tests" as well to REALLY test everything end to end. Not necessarily as a test at each commit/PR/master etc, but as a weekly/occasionally triggered CRON/manual run to sanity check end to end. The reason being they are at least 1h+ long and may require manual results inspection.
